### PR TITLE
Msgfplus tsv showDecoy

### PR DESCRIPTION
**Pull request by glormph**
This implements the `-showDecoy 1` parameter for creating TSV files from MSGF+ mzids. Enabling the parameter makes the resulting TSV file include decoy PSMs.

### CODE REVIEW

File: tools/msgfplus/msgfplus.xml [hunk 1]
```diff
@@ -82,7 +85,13 @@
                       label="Collection: Pairs of spectra (forward) and FASTA database (reverse)"/>
               </when>
         </conditional>
-        <param name="tsvcheck" type="boolean" label="Output TSV as well?" />
+        <conditional name="tsv">
```
Comment (bgruening) [inline tools/msgfplus/msgfplus.xml]: Why do you use a conditional here, it could be a single boolean isn't it?

Comment (glormph) [reply tools/msgfplus/msgfplus.xml]: It's to make the showDecoy boolean appear only if produce TSV is checked. Wrong UI?

Comment (bgruening) [reply tools/msgfplus/msgfplus.xml]: Ok got it, I think this is a little bit confusing. Why not a simple select, "TSV outpout", "TSV ooutput with decoys" ...

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**